### PR TITLE
included source .proto files from ISIS, regenerated _pb2 files with p…

### LINKED ISSUE #210: Wrong libprotobuf Version Dependency?
I'm doing a new install and was running into issues for any of the available PLIO versions (1.5.2–1.5.5).  I install it, then open Python and do something simple like `import plio` and the following results:

```Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/anaconda3/envs/plio/lib/python3.12/site-packages/plio/__init__.py", line 17, in <module>
    from . import io
  File "/opt/anaconda3/envs/plio/lib/python3.12/site-packages/plio/io/__init__.py", line 22, in <module>
    from . import io_controlnetwork
  File "/opt/anaconda3/envs/plio/lib/python3.12/site-packages/plio/io/io_controlnetwork.py", line 10, in <module>
    from plio.io import ControlNetFileV0002_pb2 as cnf
  File "/opt/anaconda3/envs/plio/lib/python3.12/site-packages/plio/io/ControlNetFileV0002_pb2.py", line 34, in <module>
    _descriptor.EnumValueDescriptor(
  File "/opt/anaconda3/envs/plio/lib/python3.12/site-packages/google/protobuf/descriptor.py", line 920, in __new__
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

I exited Python, ran `conda install protobuf=3.20`, and now it seems to be working -- or, at least, doesn't throw errors when I just run `import plio`.  The version that was installed originally was 5.27.5.  I haven't tested the rest of my code so I'm not sure if this relatively simple work-around solved things, but it's something to look into.  Note that this is on an M1-chip Mac -- I've been migrating to arm-64 architecture for all my conda installs except ISIS to try to avoid the Rosetta 2 translation hit.

### PR DESCRIPTION
**Pull request by jrcain-usgs**
included source .proto files from ISIS, regenerated _pb2 files with proton

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

Fixes #210

### CODE REVIEW

**Comment by Kelvinrr** [comment]
we can probably ignore 3.8, probably save to remove 3.8 tests altogether.

**Comment by Kelvinrr** [comment]
Add changelog entry and reference the issue this addresses. I imagine the existing io_controlnetork tests would catch any protobuf failures due to using latest version

**Comment by jrcain-usgs** [comment]
> we can probably ignore 3.8, probably save to remove 3.8 tests altogether.

Removed 3.8 and added 3.12, think I should worry about python 3.13?

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ release.
 
 ### Fixed
 - Fixed a bug where scale and offset were being applied backwards to GeoDataset objects.
+- Copied and pieced together .proto files from ISIS3/src/control/objs/ControlNetVersioner to plio/plio/io. 
+- Regenerated _pb2.py files with protoc 28.2 to fix error with old-protoc-generated files. [#210](https://github.com/DOI-USGS/plio/issues/210)
```
Comment (Kelvinrr) [inline CHANGELOG.md:44]: This should be one changelog entry

**Review by Kelvinrr — APPROVED** [review]

**Pull request merged**